docs/documentation/tabs-explained/temperature-and-humidity/relative-humidity-explained.md

```
@@ -31,10 +31,12 @@ The four heatmaps give a clear idea of the comparison of the four climate types.
 The last tool for relative humidity assessment is the statistics table. The earlier graphically made evaluations can be supported by the numbers. The following are listed, for each month:
 
 * the relative humidity means;
-* the [standard deviations](https://en.wikipedia.org/wiki/Standard\_deviation);
+* the [standard deviations](https://en.wikipedia.org/wiki/Standard_deviation);
 * the minimum values;
 * the [percentiles values](https://en.wikipedia.org/wiki/Percentile) (1%, 25%, 50%, 75%, 99%);
 * the maximum values.
 *
 
-    <figure><img src="../../../.gitbook/assets/Desc stat RH.png" alt=""><figcaption><p>Descriptive statistics of relative humidity trend in a temperate climate, Berkeley (USA)</p></figcaption></figure>
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)